gen_topo.sh

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env sh
+#PBS -P tm70
+#PBS -q normal
+#PBS -l walltime=4:00:00,mem=4GB
+#PBS -l wd
+#PBS -lstorage=gdata/hh5+gdata/ik11+gdata/tm70
+
+module use /g/data/hh5/public/modules
+module load conda/analysis3
+
+# Build domain-tools
+./build.sh
+
+set -x
+set -e
+
+cp -L --preserve=timestamps /g/data/tm70/ek4684/New_grid_input_files_025deg_75zlevels/ocean_hgrid.nc .
+cp -L --preserve=timestamps /g/data/tm70/ek4684/New_grid_input_files_025deg_75zlevels/ocean_vgrid.nc .
+ln -sf /g/data/ik11/inputs/GEBCO_2024/GEBCO_2024.nc ./
+
+# Convert double precision vgrid to single
+./domain-tools/bin/float_vgrid --vgrid ocean_vgrid.nc --vgrid_type mom6
+
+# Interpolate topography on horizontal grid:
+./domain-tools/bin/topogtools gen_topo -i GEBCO_2024.nc -o topog_new.nc --hgrid ocean_hgrid.nc --tripolar --longitude-offset -100 
```

Comment:
I think it would be clearer if all the intermediate `topo_` files went into a seperate folder and only the final `topog.nc` file is kept in the main folder

Reply:
Added in this f77f4f2